### PR TITLE
Disable proxy_buffering for staging server.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: roles/nginx/templates/server.conf.j2 [hunk 1]
```diff
@@ -42,6 +42,10 @@ server {
         proxy_pass  http://127.0.0.1:{{ vhost.local_port }};
         {% endif -%}
 
+        {% if vhost.proxy_buffering is defined -%}
+        proxy_buffering off;
```
Comment (ghost) [inline roles/nginx/templates/server.conf.j2]: or i need to do it like proxy_buffering {{ vhost.proxy_buffering }} ?

Comment (mblomdahl) [reply roles/nginx/templates/server.conf.j2]: You could either have

    proxy_buffering {{ 'off' if not vhost.proxy_buffering else 'on' }};

So, then we know that, if there is a variable about buffering, then we'll add a statement in the server block about it. And if it's something truthy then we write "on", if it's something falsy then "off".

**Review by mblomdahl — COMMENTED** [review]
Haapy to see you're making progress! :)

**Review by mblomdahl — APPROVED** [review]
Can I run the provisioning now?

**Comment by mblomdahl** [comment]
@vtepliuk  Looks like it will have the intended effect, and no adverse side effects:

Screenshot 2019-05-08 at 15 00 36

**Comment by ghost** [comment]
> Can I run the provisioning now?

Yes, you can.

**Comment by mblomdahl** [comment]
@vtepliuk Done!

```plain
(venv) EMB2:hq.mkdevops.se mblomdahl$ ansible-playbook bootstrap.yml --start-at-task="template /etc/nginx/conf.d/server.conf" --diff  -K
SUDO password: 

PLAY [bootstrap] ***************************************************************************************************************************************************************************************

TASK [Gathering Facts gather_subset=['all'], gather_timeout=10] ****************************************************************************************************************************************
ok: [hq.mkdevops.se]

TASK [nginx : template /etc/nginx/conf.d/server.conf src=server.conf.j2, dest=/etc/nginx/conf.d/server.conf] *******************************************************************************************
--- before: /etc/nginx/conf.d/server.conf
+++ after: /Users/mblomdahl/.ansible/tmp/ansible-local-18168kezhl2tr/tmpl8awv21v/server.conf.j2
@@ -248,7 +248,7 @@
     charset        utf-8;
     gzip           on;
 
-    client_max_body_size 128M;
+    client_max_body_size 32M;
     
     ssl                  on;
     ssl_certificate      /etc/letsencrypt/live/staging.stockholmsaltspa.com/fullchain.pem;
@@ -262,6 +262,7 @@
     
     location / {
         proxy_pass  http://127.0.0.1:8091;
+        proxy_buffering off;
         root /usr/share/nginx/html/staging.stockholmsaltspa.com;
         autoindex on;
     }

changed: [hq.mkdevops.se]

TASK [nginx : nginx running and enabled name=nginx, state=started, enabled=yes] ************************************************************************************************************************
ok: [hq.mkdevops.se]

TASK [docker : docker-ce repo name=docker-ce-stable, description=Docker CE Stable - $basearch, baseurl=https://download.docker.com/linux/centos/7/$basearch/stable, gpgkey=https://download.docker.com/linux/centos/gpg, enabled=False] ***
ok: [hq.mkdevops.se]

TASK [docker : install docker-ce-18.06.1.ce from docker-ce name=docker-ce-18.06.1.ce, enablerepo=docker-ce-stable, state=present] **********************************************************************
ok: [hq.mkdevops.se]

TASK [docker : install docker-compose-1.23.1 name=docker-compose, version=1.23.1, state=present] *******************************************************************************************************
ok: [hq.mkdevops.se]

TASK [docker : create 144g var_lib_docker for /var/lib/docker vg=sys, lv=var_lib_docker, size=144g, resizefs=yes] **************************************************************************************
ok: [hq.mkdevops.se]

TASK [docker : format var_lib_docker as xfs fstype=xfs, dev=/dev/sys/var_lib_docker, resizefs=True, opts=-n ftype=1] ***********************************************************************************
skipping: [hq.mkdevops.se]

TASK [docker : mount var_lib_docker as /var/lib/docker path=/var/lib/docker, fstype=xfs, src=/dev/sys/var_lib_docker, state=mounted, boot=True] ********************************************************
ok: [hq.mkdevops.se]

TASK [docker : create /etc/docker dir path=/etc/docker, owner=root, group=docker, state=directory] *****************************************************************************************************
ok: [hq.mkdevops.se]

TASK [docker : template docker daemon.json src=daemon.json.j2, dest=/etc/docker/daemon.json, owner=root, group=docker] *********************************************************************************
ok: [hq.mkdevops.se]

TASK [docker : docker (re)started and enabled name=docker, state={{ 're' if docker_daemon_json is changed else '' }}started, enabled=yes] **************************************************************
ok: [hq.mkdevops.se]

RUNNING HANDLER [nginx : restart nginx name=nginx, state=restarted] ************************************************************************************************************************************
changed: [hq.mkdevops.se]

PLAY RECAP *********************************************************************************************************************************************************************************************
hq.mkdevops.se             : ok=12   changed=2    unreachable=0    failed=0   

```

**Pull request merged**